src/oidc/static/pages-source.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
```

Comment:
Design: https://www.figma.com/file/KNSE9YVqWCNJaSeIiOFMFo/OIDC-Auth-in-Compass?t=8yfhhLQxkYruuYDE-0

<details><summary>Screenshots the fold</summary>

![Screenshot 2023-03-02 at 19-03-39 Login Successful](https://user-images.githubusercontent.com/899444/222513662-b986453f-69e1-4918-9947-687e2cec7936.png)
![Screenshot 2023-03-02 at 19-03-43 Page Not Found](https://user-images.githubusercontent.com/899444/222513664-54986447-3747-45ea-9c93-d9892b25ee51.png)
![Screenshot 2023-03-02 at 19-03-48 Authentication Failed](https://user-images.githubusercontent.com/899444/222513666-f9277464-1507-49e2-9c1a-0ba9d7a16fbe.png)
![Screenshot 2023-03-02 at 19-03-51 Authentication Failed](https://user-images.githubusercontent.com/899444/222513668-cfbd5712-3ad6-4d98-8f07-9e51cd8de6da.png)
![Screenshot 2023-03-02 at 19-03-55 Authentication Failed](https://user-images.githubusercontent.com/899444/222513671-a925a01f-df39-4971-974c-00d30fe1c39f.png)
![Screenshot 2023-03-02 at 19-03-58 Authentication Failed](https://user-images.githubusercontent.com/899444/222513674-76a092f1-51e7-49d0-9004-df8ade5d3955.png)
![Screenshot 2023-03-02 at 19-04-02 Authentication Failed](https://user-images.githubusercontent.com/899444/222513677-ee0e33cb-3245-4825-b199-82bc831e53e5.png)


</details>